ArdalisRating.Tests/RatingEngineRate.cs

```diff
@@ -0,0 +1,48 @@
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using Xunit;
+
+namespace ArdalisRating.Tests
+{
+    public class RatingEngineRate
+    {
+[Fact]
+public void ReturnsRatingOf10000For200000LandPolicy()
+{
+    var policy = new Policy
+    {
+        Type = PolicyType.Land,
+        BondAmount = 200000,
+        Valuation = 200000
+    };
+    string json = JsonConvert.SerializeObject(policy);
+    File.WriteAllText("policy.json", json);
+
+    var engine = new RatingEngine();
+    engine.Rate();
+    var result = engine.Rating;
+
+    Assert.Equal(10000, result);
+}
+
+        [Fact]
+        public void ReturnsRatingOf0For200000BondOn260000LandPolicy()
+        {
+            var policy = new Policy
+            {
+                Type = PolicyType.Land,
+                BondAmount = 200000,
+                Valuation = 260000
+            };
+            string json = JsonConvert.SerializeObject(policy);
+            File.WriteAllText("policy.json", json);
+
+            var engine = new RatingEngine();
+            engine.Rate();
+            var result = engine.Rating;
+
+            Assert.Equal(0, result);
+        }
+    }
+}
```

Comment:
All these tests is doing the same act, for decreasing the test code and using duplicates you can create special test method which can do the same act with different test data and use it in each test. But that will not be clear to understanding, for example if you will create 100 test case with different data. Fortunately xUnit have special attribute which called [InlineData()] and using with other attribute [Theory]  that gives us posibility to create one test with a lot of diffrent tests data,  you can read about DataSource and Data-driven Testing using XUnit and use this approach to your tests